pkg/config/output.go

```diff
@@ -577,3 +577,11 @@ func redactStreamKeys(stream *livekit.StreamOutput) {
 		}
 	}
 }
+
+func clean(filepath string) string {
```

Comment:
It looks like `path.Clean` will not remove `../` at the beginning of a relative path (it will for an absolute path, such as `/..`). Unless we are already enforcing this somewhere else, should be make sure that the resulting output doesn't start with `..`?